app/src/main/java/edu/nextstep/camp/calculator/HistoryAdapter.kt

```diff
@@ -0,0 +1,44 @@
+package edu.nextstep.camp.calculator
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import edu.nextstep.camp.calculator.databinding.ItemResultBinding
+import edu.nextstep.camp.calculator.domain.History
+
+/**
+ * Created by link.js on 2022. 07. 23..
+ */
+class HistoryAdapter : ListAdapter<History, HistoryViewHolder>(diffUtil) {
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): HistoryViewHolder {
+        return HistoryViewHolder(ItemResultBinding.inflate(LayoutInflater.from(parent.context),
+            parent,
+            false))
+    }
```

Comment:
인텐트는 아래 처럼 파라미터가 어디에 들어가는 지 정렬해보는 것은 어떨까요?
```suggestion
        return HistoryViewHolder(
            ItemResultBinding.inflate(
                LayoutInflater.from(parent.context), parent, false)
            )
        )
    }
```